### ISSUE TITLE
error: Undefined name ‘ImageConstants’

### ISSUE THREAD

**Issue by UmaiZ** [inspireui/support#2816]
Follow the documents step 
Open the fluxstore product, copy and rename the lib/example/woocommerce/config_localhost.dart and lib/example/woocommerce/config_localhost.json to lib/common/config.dart lib/config/config_en.json

But after copy paste file there are many errors in different file like this

error: Undefined name ‘ImageConstants’. (undefined_identifier at [fstore] lib\common\config.dart:167) error: Undefined name ‘kAdType’. (undefined_identifier at [fstore] lib\models\advertisement.dart:23) error: Case expressions must be constant. (non_constant_case_expression at [fstore] lib\models\advertisement.dart:23)

**Comment by minhcasi** [comment]
@UmaiZ - please try to fix the import as below code

```
export 'config/advertise.dart';
export 'config/general.dart';
export 'config/onboarding.dart';
export 'config/payments.dart';
export 'config/products.dart';
export 'config/smartchat.dart';

/// Server config REAL
const serverConfig = {
  "type": "woo",
  "url": "http://mstore.local",
  "consumerKey": "ck_98f9ca71c82ec652ac27194eafef4a9cf2af300a",
  "consumerSecret": "cs_83d385c0711ace08304126f48618d7a9aa7ff663",
  "blog": "http://demo.mstore.io", //Your website woocommerce. You can remove this line if it same url
  "forgetPassword": "http://demo.mstore.io/wp-login.php?action=lostpassword"
};

```

**Comment by UmaiZ** [comment]
But products are not showing i add my URL consumer key and secret. Also enable apis etc

**Comment by UmaiZ** [comment]
> @UmaiZ - please try to fix the import as below code
> 
> ```
> export 'config/advertise.dart';
> export 'config/general.dart';
> export 'config/onboarding.dart';
> export 'config/payments.dart';
> export 'config/products.dart';
> export 'config/smartchat.dart';
> 
> /// Server config REAL
> const serverConfig = {
>   "type": "woo",
>   "url": "http://mstore.local",
>   "consumerKey": "ck_98f9ca71c82ec652ac27194eafef4a9cf2af300a",
>   "consumerSecret": "cs_83d385c0711ace08304126f48618d7a9aa7ff663",
>   "blog": "http://demo.mstore.io", //Your website woocommerce. You can remove this line if it same url
>   "forgetPassword": "http://demo.mstore.io/wp-login.php?action=lostpassword"
> };
> ```

**Comment by UmaiZ** [comment]
Sorry bymistake closed :D

**Comment by leo0360** [comment]
I got same issue!!

copied "lib/example/woocommerce/config_localhost.dart" has below import,
  import 'constants/image_constants.dart';
which giving error, no such file(image_constants.dart) in this path

**Comment by minhcasi** [comment]
Hi, please use this example 

```
export 'config/advertise.dart';
export 'config/general.dart';
export 'config/onboarding.dart';
export 'config/payments.dart';
export 'config/products.dart';
export 'config/smartchat.dart';

/// Server config REAL
const serverConfig = {
  "type": "woo",
  "url": "http://mstore.local",
  "consumerKey": "ck_98f9ca71c82ec652ac27194eafef4a9cf2af300a",
  "consumerSecret": "cs_83d385c0711ace08304126f48618d7a9aa7ff663",
  "blog": "http://demo.mstore.io", //Your website woocommerce. You can remove this line if it same url
  "forgetPassword": "http://demo.mstore.io/wp-login.php?action=lostpassword"
};
```

We also fix on the latest source 1.6.7 on Gitlab or it will be available in 1-2 days - thank you!
